src/components/Label/Label.tsx

```diff
@@ -162,32 +175,37 @@ const RightIcon = styled(Icon)<IStyledLabel>`
 const Label: ILabel = props => {
   const { className, style, ...rest } = props
   const { text, icons, insetColor } = rest
+  /**
+   * Styled Component theme should be needed here, because getFontColor take theme
```

Comment:
But `getFontColor` color does not seem to take the theme argument...

Reply:
It is passed in line `scProps` object.